src/pages/TeamSettings/CloudHooks.vue

```diff
@@ -78,8 +78,12 @@ export default {
   },
   computed: {
     ...mapGetters('tenant', ['tenant', 'role']),
+    ...mapGetters('license', ['hasPermission']),
     isReadOnlyUser() {
-      return this.role === 'READ_ONLY_USER'
+      return (
+        !this.hasPermission('create', 'role') &&
+        !this.hasPermission('create', 'cloud-hook')
```

Comment:
What's the purpose of the `create:role` check here? We'll want to break out the cloud hook CRUD permissions based on the operations the user can take, i.e. `create:cloud-hook` for creating new ones, create and delete for editing (since this isn't a true edit/update) etc